### ISSUE TITLE
dnsimple_lets_encrypt_certificate not working with provider version 1.1.1

### ISSUE THREAD

**Issue by ehaselwanter** [dnsimple/terraform-provider-dnsimple#109]
### Terraform Version

```
Terraform v1.4.6
on linux_amd64
```

### Affected Resource(s)

- dnsimple_lets_encrypt_certificate

### Terraform Configuration Files
```hcl
resource "dnsimple_lets_encrypt_certificate" "esx-4-dc1-cert" {
  domain_id  = dnsimple_domain.my_cloud.id
  auto_renew = true
  signature_algorithm = "RSA"
  name       = "esx-4.dc1"
}
```

### Expected Behavior

cert is created

### Actual Behavior

```
╷
│ Error: Value Conversion Error
│ 
│   with dnsimple_lets_encrypt_certificate.esx-4-dc1-cert,
│ An unexpected error was encountered trying to convert the value. This is
│ always an error in the provider. Please report the following to the
│ provider developer:
│ 
│ Path: 
│ Error: target must be a pointer, got []string, which is a slice
```

**Comment by DXTimer** [comment]
Hi @ehaselwanter thank you for reporting. We were able to reproduce the behavior and have already started working on a fix.

**Comment by wilson-dnsimple** [comment]
Thanks for raising the issue @ehaselwanter, we've released a new version 1.1.2 that should resolve the issue.
